Vocable/AppDelegate.swift

```diff
@@ -177,10 +176,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let request: NSFetchRequest<Category> = Category.fetchRequest()
         request.predicate = {
             let identifiers = Set(presets.categories.map { $0.id })
```

Comment:
You updated the closure to a keypath above. Could we do the same here?

```suggestion
            let identifiers = Set(presets.categories.map(\.id))
```